lib/text/string.effekt

```diff
@@ -28,4 +28,7 @@ extern pure def unsafeCharAt(str: String)(n: Int): String =
   "str[n]"
 
 extern pure def unsafeToInt(str: String): Int =
-  "Number.isNaN(parseInt(str)) ? undefined : parseInt(str)"
\ No newline at end of file
+  "Number.isNaN(parseInt(str)) ? undefined : parseInt(str)"
+
+extern pure def concat(str1: String, str2: String): String =
+  "str1.concat(str2)"
```

Comment:
It is great that you add what you need, but this is already defined in the prelude :) 

https://github.com/effekt-lang/effekt/blob/98ba53335e8fa53c9ebc72f5fc18142a2fc52457/lib/effekt.effekt#L37-L38